sdk/src/main/java/com/desk/android/sdk/config/ContactUsPropertyConfig.java

```diff
@@ -217,7 +236,59 @@ public String getCallUsPhoneNumber() {
      */
     @Override
     public String getCallUsPhoneNumber(int brandId) {
-        return getString(properties, getCallUsPhoneNumber(), buildBrandKey(KEY_CONTACT_US_PHONE_NUMBER), brandId);
+        return getStringWithArgs(buildBrandKey(KEY_CONTACT_US_PHONE_NUMBER), getCallUsPhoneNumber(), properties, brandId);
+    }
+
+    @Override
+    @NonNull
+    public List<String> getCustomFieldKeys() {
+        return getCustomFieldKeys(KEY_CONTACT_US_CUSTOM_FIELD_KEYS);
+    }
+
+    @Override
+    @NonNull
+    public List<String> getCustomFieldKeys(int brandId) {
+        return getCustomFieldKeys(buildBrandKey(KEY_CONTACT_US_CUSTOM_FIELD_KEYS), brandId);
+    }
+
+    @Override
+    public HashMap<String, CustomFieldProperties> getCustomFieldProperties() {
+        List<String> keys = getCustomFieldKeys();
+        HashMap<String, CustomFieldProperties> customFields = new HashMap<>(keys.size());
+        if (keys.size() > 0) {
+            for (String key : keys) {
+                String value = getStringWithArgs(KEY_CONTACT_US_CUSTOM_FIELD_VALUE, properties, key.trim());
+                CustomFieldProperties properties = new CustomFieldProperties.Builder(key).value(value).create();
+                customFields.put(key, properties);
+            }
+        }
+        return customFields;
+    }
+
+    @Override
+    public HashMap<String, CustomFieldProperties> getCustomFieldProperties(int brandId) {
+        List<String> keys = getCustomFieldKeys(brandId);
```

Comment:
I think this method and the one above can be reduced to one method; they seem to share much of the same boilerplate code. A possible enhancement for a future release.